migrate/migrate.go

```diff
@@ -18,11 +18,11 @@
 
 package migrate
 
-import (
-	"fmt"
+// TODO: Remove old migrators and create migrator from stable playground to playground v2
```

Comment:
I suggest either implementing this as part of this PR or making it a separate PR, but in any case don't leave commented out code.

Reply:
I started another PR to create the migrator, it's planned for the next sprint.